### ISSUE TITLE
Wrong system temperature in position switching

### ISSUE THREAD

**Issue by astrofle** [GreenBankObservatory/dysh#37]
When using `dysh.fits.gbtfitsload.GBTFITSLoad.getps` the resulting `dysh.spectra.scan.GBTPSScan` produces wrong system temperatures after calling `calibrate`.
Example:
```
from dysh.fits.gbtfitsload import GBTFITSLoad
# Assuming you are in the dysh root dir.
sdf_file = "dysh/src/dysh/fits/tests/data/TGBT21A_501_11_ifnum_0_int_0-2.fits"
sdf = GBTFITSLoad(sdf_file)
psscan = sdf.getps(152, plnum=0)
psscan.calibrate()
psscan.tsys
>>> array([17.17140405, 17.3474724 , 17.0515161 ])
# The above should be:
>>> array([17.24000345, 17.17140405, 17.15663698])
```
The file used for this is in the `pedro-devel` branch.

**Comment by astrofle** [comment]
This arises because the rows are not ordered!
```
psscan._refonrows, psscan._refoffrows
>>> ([19, 23, 15], [18, 14, 22])
```
So `dysh` is mixing up integrations (notice how 23 is paired with 14 and 15 with 22). 
Interesting that this makes a difference given that physically it should not matter.

**Comment by mpound** [comment]
Good sleuthing!

**Comment by mpound** [comment]
fixed in mwp-devel with 0126212
